sql/_13_issues/_12_1h/cases/bug_bts_6835.sql

```diff
@@ -59,5 +59,5 @@ drop t;
 
 create table t (a int, b int);
 -- should give an error
-create index idx_invalid on t(if(a,b,-b));
-drop t;
\ No newline at end of file
+create index idx_invalid on t(if(a<>0,b<>0,-b<>0));
```

Comment:
Only the first of the if function needs to be a logical expression.
`create index idx_invalid on t(if(a<>0,b,-b));`